.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: false
+language: python
+install: pip install tox
+matrix:
+  include:
```

Comment:
Why have we lost the Py3 test configuration?  The internal job for the legacy alt-src repo did cover Python3 (all xfails, but that's still better than no CI at all).

Reply:
Well, it was suggested because it's not doing much at the moment and could always be added when necessary. Though, now that I think of it, it wasn't totally worthless as it was ensuring that those tests *do* fail on Python3, so we'd notice and remove the xfail when that changes.

Reply:
Adding python3.6 back.